canonical-web-and-design/docs.ubuntu.com
# Integrate cookie-policy v3.0.4

## Done
- Add global-nav and cookie-policy dependencies
- Built with webpack

## QA
- Load the site
- Check the global nav works as expecred
- Check the cookie policy is there and works
- Go to the footer and manage your cookies and see it opens again

## Issue / Card
Fixes https://github.com/canonical-web-and-design/docs.ubuntu.com/issues/268
